libs/stdlib/iec61131-st/trunc_int.st

```diff
@@ -0,0 +1,320 @@
+
+(********************
+*
+* Converts a floating point number to SINT
+*
+********************)
+FUNCTION TRUNC_SINT<T: ANY_REAL> : SINT
+    VAR_INPUT
+        in : T;
+    END_VAR
+END_FUNCTION
+
+
+(********************
+*
+* Converts REAL to SINT
+*
+********************)
+FUNCTION TRUNC_SINT__REAL : SINT
+    VAR_INPUT
+        in : REAL;
+    END_VAR
+
+    TRUNC_SINT__REAL := in;
+END_FUNCTION
+
+
+(********************
```

Comment:
nit - double newlines
I might have missed this in the other PRs aswell, but not a blocker.

Reply:
I'll address this (and moving the Python files into the utils/ folder) in the upcoming PRs